src/main/java/gui/panes/ManualConfigPane.java

```diff
@@ -0,0 +1,370 @@
+package gui.panes;
+
+import gui.controllers.RoomController;
+import gui.controllers.VideoController;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.MenuItem;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.GridPane;
+import javafx.scene.text.Font;
+import javafx.stage.Stage;
+
+import java.util.ArrayList;
+
+/**
+ * Class that creates the ManualConfigPane in a new Scene.
+ */
+public class ManualConfigPane {
+
+    private VideoController videoController;
+    private RoomController roomController;
+    private Stage manualStage;
+    private GridPane fillInPane;
+    private Font font;
+    private final int maxWidth = 60;
+
+    /**
+     * Constructor for ManualConfigPane.
+     * @param vControl the videoController
+     * @param rControl the roomController
+     */
+    public ManualConfigPane(final VideoController vControl, final RoomController rControl) {
+        this.videoController = vControl;
+        this.roomController = rControl;
+        this.manualStage = new Stage();
+        this.fillInPane = new GridPane();
+
+        final int fontSize = 20;
```

Comment:
Font and font size can also be placed in the css file and then add a styleclass to that specific pane. May be better for separation of functionalities. But not a super big deal